python/ribasim/ribasim/schemas.py

```diff
@@ -225,6 +225,12 @@ class DiscreteControlConditionSchema(_BaseSchema):
     greater_than: Series[Annotated[pd.ArrowDtype, pyarrow.float64()]] = pa.Field(
         nullable=False
     )
+    condition_id: Series[Annotated[pd.ArrowDtype, pyarrow.int32()]] = pa.Field(
```

Comment:
Maybe this should be the first field in this schema? Or second, after node_id? Not sure here, maybe sort order?